python/cuml/dask/linear_model/logistic_regression.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2019-2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+from dask_glm.estimators import LogisticRegression as LogisticRegressionGLM
+import cupy as cp
+import dask_cudf
+
+
+class LogisticRegression(BaseEstimator):
+
+    def __init__(self, client=None, verbose=False, **kwargs):
+        super(LogisticRegression, self).__init__(client=client,
+                                                 verbose=verbose,
+                                                 **kwargs)
+
+    def fit(self, X, y):
+        """
+        Fit the model with X and y.
+
+        Parameters
+        ----------
+        X : Dask cuDF dataframe or CuPy backed Dask Array (n_rows, n_features)
+            Features for regression
+        y : Dask cuDF Series or CuPy backed Dask Array (n_rows,)
+            Label (outcome values)
+        """
+        X = self._to_dask_cupy_array(X)
+        y = self._to_dask_cupy_array(y)
+        lr = LogisticRegressionGLM(**self.kwargs)
+        lr.fit(X, y)
+        self.lr = lr
+        return self
+
+    def predict(self, X):
+        """
+        Make predictions for X and returns a dask collection.
+
+        Parameters
+        ----------
+        X : Dask cuDF dataframe  or CuPy backed Dask Array (n_rows, n_features)
+            Distributed dense matrix (floats or doubles) of shape
+            (n_samples, n_features).
+
+        Returns
+        -------
+        y : Dask cuDF Series or CuPy backed Dask Array (n_rows,)
+        """
+        X = self._to_dask_cupy_array(X)
+        return self.lr.predict(X)
+
+    def _to_dask_cupy_array(self, X):
```

Comment:
This function might be better in https://github.com/rapidsai/cuml/blob/branch-0.19/python/cuml/dask/common/input_utils.py#L195 named something like `dask_cudf_to_dask_cupy`

Reply:
Do you anticipate wanting to use something like this elsewhere in the code? From what I can tell, most of these input utilities seem to be oriented toward raw futures (`to_dask_cupy`, `_to_dask_cudf`, etc.).

If we assume the data inputs to this implementation are instead dask collections (explicitly dataframe or array), do you think it's worth adding to this utility file? I'm slightly concerned about causing adding functions that with similar names and causing confusion

Reply:
For now, I've updated this function and left it in this class.